panellayout/src/main/java/com/wayfair/panellayout/PanelLayout.kt

```diff
@@ -895,6 +895,7 @@ class PanelLayout @JvmOverloads constructor(
 
         override fun onTouch(v: View, event: MotionEvent): Boolean {
             return when (event.action) {
+                ACTION_DOWN -> true
```

Comment:
what does this mean ? Why was it causing a bug ?

Reply:
Something similar was there also with the bug fix from last week before we deployed the alpha version.